services/galley/src/Galley/API/Teams.hs

```diff
@@ -532,21 +542,21 @@ getBindingTeamId zusr = withBindingTeam zusr $ pure . json
 
 getBindingTeamMembers :: UserId -> Galley Response
 getBindingTeamMembers zusr = withBindingTeam zusr $ \tid -> do
-  members <- Data.teamMembers tid
-  pure . json $ newTeamMemberList members
+  members <- Data.teamMembersUnsafeForLargeTeams tid
+  pure . json $ newTeamMemberList members False
```

Comment:
This is going to bite you when you're merging this to develop.  I guess it's good to rebase occasionally if a branch is open for longer, then you would have made these changes on top of a more recent develop.